go/cmd/remotetool/main.go

```diff
@@ -115,6 +117,14 @@ func main() {
 			log.Exitf("error reexecuting action %v: %v", *digest, err)
 		}
 
+	case uploadBlob:
+		if *pathPrefix == "" {
+			log.Exitf("--path must be specified.")
+		}
+		if err := c.UploadBlob(ctx, *digest, *pathPrefix); err != nil {
```

Comment:
Don't pass the digest from here, compute and print it from the file contents inside the function.

Reply:
done